### DISCUSSION TITLE
Drag on key press

### DISCUSSION THREAD

**Discussion by JulesDebeaumont** [bcakmakoglu/vue-flow#1333] [Q&A] [answered]
Hello ! 

I've look around the docs, and didnt find an intended way to drag a node by using a key press.
Like, instead of left-click, hold the click to move a node, and then release, I would like for example to select a node (I found the props),
press a key (the G key, kind of the Blender shortcut for Grab), move the node, and then press Enter to exit drag mode.
Is there a proper way to do this ?
I've tried to enable the 'dragging' props on a selected node, thinking that the node would follow the cursor, even if I didnt hold the click.

Also thanks for the team for making and maintaining this, I've been using it for a year now and its absolutly amazing! 🚀🧡

**Comment by bcakmakoglu** [comment, marked as answer]
There no built-in functionality to toggle draggability but it's possible by handling the key press yourself.

There is a prop called `nodesDraggable` that you can use to toggle draggability of nodes, just toggle it the keypress you're listening to 😄
